cosmos/providers/dbt/core/operators.py

```diff
@@ -127,7 +128,11 @@ def __init__(
         self.output_encoding = output_encoding
         self.skip_exit_code = skip_exit_code
         self.cancel_query_on_kill = cancel_query_on_kill
-        self.dbt_executable_path = dbt_executable_path
+        dbt_ol_path = shutil.which("dbt-ol")
```

Comment:
Can you add a comment mentioning that `dbt-ol` refers to `dbt open-lineage.` might make it easier for outside users to understand what's going on.